cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -556,18 +556,18 @@ def replace_certificates(self):
                 os.path.exists(constants.NEW_POSTGRESQL_CA_CERT_FILE_PATH)):
             self.log_replacing_certificates()
             self._write_certs_to_config()
-            if config[POSTGRESQL_SERVER]['cluster']['nodes']:  # cluster case
-                self.handle_cluster_certificates()
-                self._restart_etcd()
-                service.restart('patroni', append_prefix=False)
-                service.verify_alive('patroni', append_prefix=False)
-
-            else:  # AIO case
+            if common.is_all_in_one_manager():  # AIO case
```

Comment:
I don't think the comment here or on the else add much- the naming of the function makes it clear.

Reply:
Correct, I'll delete the comments.